src/Kafka/ConsumerGroups/components/ConsumerGroupDetails/ConsumerGroupByKafka.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Flex,
+  FlexItem,
+  Stack,
+  TextContent,
+  TextVariants,
+  Text,
+} from "@patternfly/react-core";
+import {
+  TableComposable,
+  TableVariant,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { ConsumerGroup } from "../../types";
+import { displayConsumerGroupState } from "../../utils";
+import { ConsumerGroupPopover } from "../ConsumerGroupPopover";
+
+export type ConsumerGroupByKafka = {
+  consumerGroupDetail: ConsumerGroup;
```

Comment:
I understand it's easy to copy-paste the type from the sdk and use it here, but it's also a way to get around of the rule of no sdk dependency. 

Instead of an object here, I'd have specified many props, one for each visual element that needs to be displayed.